client/src/app/KeyboardBindings.js

```diff
@@ -175,7 +198,29 @@ export default class KeyboardBindings {
     this.undo = findUndo(menu);
     this.redo = findRedo(menu);
 
+    menu = this.updateRemoveSelectionEntry(menu);

```

Comment:
We should do this earlier, as it is confusing otherwise => previous steps extract entries from the old, soon gone menu.